projects/adrv9371x/common/adrv9371x_bd.tcl

```diff
@@ -262,6 +262,7 @@ ad_connect  logic_or/Op2  tx_fir_interpolator/valid_out_2
 ad_connect  logic_or/Res  util_ad9371_tx_upack/fifo_rd_en
 
 ad_connect  tx_fir_interpolator/active dac_fir_filter_active
+ad_connect  tx_fir_interpolator/VCC    VCC
```

Comment:
Why is this not needed in the 9009 project ? It should have the same interpolator instantiated.

Reply:
I agree with Laci, we can safely ignore the first two commits.